text/text.go

```diff
@@ -297,6 +300,59 @@ func Draw(dst *ebiten.Image, text string, face font.Face, x, y int, clr color.Co
 
 		prevR = r
 	}
+}
+
+// MeasureString returns the measured size of a given string using a given font.
```

Comment:
Please add a comment about how to treat new lines (Metrics().Height of the given font is used).

Reply:
I'm not sure I understand. MeasureString returns a Point with X *and* Y values, implying that both width and height are correctly measured.

Reply:
Considering multiple lines is an important feature of this function. It is not trivial to users what line-height is adopted.

Reply:
I can't come up with a fitting comment that doesn't also sound really stilted. 

This method does all the right things automatically. The user shouldn't have to worry about how newlines work because they are already familiar with how they work from the Draw method.
If I explain in the comment how the method handles newlines, I'd also have to explain how it handles special characters, multiple font faces, ascenders, descenders, and so on.

Reply:
I agree you suggestion makes sense, thanks!

Reply:
Could you add the comment you suggested?